cammac60/MacRae-Photo
# Splash comp

### What does this PR do? 

- Creates the splash page component. 
- Adds linking functionality to link to other pages and ig.

----------

### How to manually test:

- `git fetch`
- `git checkout splash-comp`
- `npm start` to ensure that then app loads on localhost with no errors.
- `npm test` to ensure that tests are still passing. 

----------

### Tickets: 

- Resolves #10 
-Resolves #12
